goals.md

```diff
@@ -0,0 +1,19 @@
+We believe all audio needs can be shared by one base. Currently there are
+multiple audio engines. They were all made with different goals in mind.
+
+# Why
+- eases the burden on maintainers.
+- speeds up development.
+- lessens the chance of a project being abandoned.
+
+# How
+We hope to achieve this:
+- demonstrating an existing crate as a good base.
+- building a crate on top of that existing crate to serve other use cases.
+- combining the best ideas and code from existing projects.
+
+# What
+- a crate being a good base: We can implement all the needed use cases with
```

Comment:
What are the core use cases? Who wants to use audio crates in Rust? Are there key tradeoffs in the space?

Reply:
Yes we should have those here. I have some answers to those questions but they are bound to be incorrect. I propose we 

- collect all possible use cases. If possible with some user story/context around those. To that end I have added what I know from Rodio.
- document the tradeoffs
- select which to focus on (the core use case).